_episodes/30-batch-system.md

```diff
@@ -1,7 +1,83 @@
 ---
 title: "Using the Batch System 101"
+teaching: 25
+exercises: 20
 questions:
 - "To be filled."
 keypoints: 
 - "To be filled." 
 ---
+
+# What is a scheduler and why do we need it?
+
+An HPC system might have thousands of nodes and thousands of users.
+How do we decide who gets what and when?
+How do we ensure that a task is run with the resources it needs?
+This job is handled by a special piece of software called the scheduler.
+On an HPC system, the scheduler manages which jobs run where and when.
+
+**here we need a simple schematic image showing what a scheduler does**
+
+- 
+
+# Working interactively
```

Comment:
By "interactively" here, are we referring to those special confgurations many HPC systems have where users are often allowed to acquire a tiny allocation of the machine and run jobs there in the same way they run jobs at the shell prompt. Or, are we really only using "interactively" as a adjective for those kinds of batch job submission cases where the job is extremely short running and and the queue is empty enough that the batch system is able to schedule and run the job so quickly it feels interactive to the user?

Reply:
we are referring to the former case, the situation where the user can get book an interactive session and run jobs on a compute node from the command line. We know that this isn't an option at many sites, and the plan was to make this section straightforward to leave out. But we wanted to keep it there for those sites where it's possible since it's a smaller conceptual step to go from own laptop to interactive node on cluster, rather than directly to batch script

Reply:
or what do you think? I'm unaware of the statistics on this, slurm is configured with this interactive option on most sites in Sweden, but I don't know how it looks globally